src/fides/api/schemas/namespace_meta/bigquery_namespace_meta.py

```diff
@@ -0,0 +1,18 @@
+from typing import Optional
+
+from fides.api.schemas.base_class import FidesSchema
+
+
+class BigQueryNamespaceMeta(FidesSchema):
```

Comment:
I'm only using this in a limited scope. It'd be nice to use this to validate datasets when we link them to a specific connection config but I'm removing this from scope for now.

Reply:
nice, that's an interesting idea. somewhat related to [my comment on the plus PR here](https://github.com/ethyca/fidesplus/pull/1621#discussion_r1769068314)